R/deprec-rerun.R

```diff
@@ -55,7 +55,7 @@ rerun <- function(.n, ...) {
   out
 }
 
-deprec_rerun <- function(.n, ...) {
+deprec_rerun <- function(.n, ..., .purrr_user_env) {
```

Comment:
```suggestion
deprec_rerun <- function(.n, ..., .purrr_user_env = caller_env(2)) {
```

Probably reasonable here since it is only called in one place? Then you don't need to set it above